Octokit.Reactive/Clients/ObservableRepositoryForksClient.cs

```diff
@@ -87,11 +116,25 @@ public IObservable<Repository> GetAll(string owner, string name, RepositoryForks
         /// <remarks>
         /// See <a href="http://developer.github.com/v3/repos/forks/#list-forks">API documentation</a> for more information.
         /// </remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="request">Used to request and filter a list of repository forks</param>
+        /// <returns></returns>
+        public IObservable<Repository> GetAll(int repositoryId, RepositoryForksListRequest request)
+        {
+            return GetAll(repositoryId, request, ApiOptions.None);
+        }
+
+        /// <summary>
+        /// Gets the list of forks defined for a repository
+        /// </summary>
+        /// <remarks>
+        /// See <a href="http://developer.github.com/v3/repos/forks/#list-forks">API documentation</a> for more information.
+        /// </remarks>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="request">Used to request and filter a list of repository forks</param>
         /// <param name="options">Options for changing the API response</param>
-        /// <returns>A <see cref="IObservable{Repository}"/> of <see cref="Repository"/>s representing forks of specified repository.</returns>
+        /// <returns></returns>
         public IObservable<Repository> GetAll(string owner, string name, RepositoryForksListRequest request, ApiOptions options)
```

Comment:
Should we allow `null` here for `RepositoryForksListRequest`? We've been restricting this elsewhere in the codebase, and it feels weird to leave this one here. We can call this out in the release notes if you think that's important...